pkg/convert/pprof/profile.go

```diff
@@ -207,6 +188,59 @@ func (p *RawProfile) Parse(ctx context.Context, putter storage.Putter, _ storage
 	return nil
 }
 
+func (p *RawProfile) ParseWithWriteBatch(c context.Context, wb stackbuilder.WriteBatchFactory, fallback storage.Putter, md ingestion.Metadata) error {
+	cont, err := p.handleRawData()
+	if err != nil || !cont {
+		return err
+	}
+	parser := streaming.NewStreamingParser(streaming.ParserConfig{
+		SpyName:       md.SpyName,
+		Labels:        md.Key.Labels(),
+		Putter:        fallback,
```

Comment:
I'm not 100% sure I understand why we need this? Is it something outside OSS?

Reply:
removed, it was a leftover from a first POC which could not do cumulative, so it used trees as fallback